presto-docs/src/main/sphinx/sql/select.rst

```diff
@@ -92,6 +92,95 @@ Additionally, the relations within a ``WITH`` clause can chain::
     relation is used. This means that if the relation is used more than once and the query
     is non-deterministic, the results may be different each time.
 
+WITH RECURSIVE Clause
+---------------------
+
+The ``WITH RECURSIVE`` clause is a variant of the ``WITH`` clause. It can define
+a list of queries to process, including recursive processing of suitable
+queries.
```

Comment:
```suggestion
The ``WITH RECURSIVE`` clause is a variant of the ``WITH`` clause. It defines
a list of named queries, possibly including recursive queries.
```

Reply:
That is too succinct imho .. I changed a bit, but left the process and recursive processing

Reply:
The fact is that the queries in the list are not "queries to process". They are just _predefined_ named queries that can be used in the main query. They are processed on demand only. That is why I opposed to using the word "process".

Reply:
Thats getting really into lower level naming .. the fact that they are parsed and evaluated is something I already consider part of processing. I think you mean processing only means when actual data is processed ... so later in the execution. My main concern really is that we say "recursive queries" which is not as explicit. Nothing about the written down query is recursive .. it only becomes a recursion when it is processed.  Anyway .. I suggest we leave it as it is.